src/unwind/unwind.hpp

```diff
@@ -1,8 +1,7 @@
 #ifndef UNWIND_HPP
 #define UNWIND_HPP
 
-#include "utils/common.hpp"
-#include "utils/utils.hpp"
+#include "cpptrace/cpptrace.hpp"
```

Comment:
```suggestion
#include <cpptrace/cpptrace.hpp>
```
(for consistency and since the intent is that it's a library header)

Reply:
Will fix